config/wikis/elwiki.yaml

```diff
@@ -0,0 +1,19 @@
+name: elwiki
+label: Greek Wikipedia
+host: el.wikipedia.org
+
+samples:
+    20k_2017:
+        quarry_url: "https://quarry.wmflabs.org/run/190663/output/0/json-lines?download=true"
+        quarry_page: "https://quarry.wmflabs.org/query/20231"
+
+unbalanced_5k_set_for_review: true
+
+default_sample: 20k_2017
+review_sample: 5k_2017
+
+models:
+    reverted:
+        tuning_params:
+            n_estimators: 500
```

Comment:
WHat happened here?

Reply:
Maybe the other params are getting pulled in as defaults?

Reply:
Yes: https://github.com/wiki-ai/editquality/blob/master/config/model_defaults.yaml